pytmx/util_pygame.py

```diff
@@ -117,16 +129,25 @@ def pygame_image_loader(filename: str, colorkey: Optional[ColorLike], **kwargs):
     pytmx image loader for pygame
 
     Parameters:
-        filename: filename, including path, to load
-        colorkey: colorkey for the image
+        filename: The filename, including path, to be loaded.
+        colorkey: The colorkey for the image.
+
+    Optional named parameters:
```

Comment:
curious why the optional parameters need to be in their own section?  there are optional parameters in another part of the code in pr which were not broken out into their own section.  also minor point of preference, periods are used when terminating sentences and not sentence *fragments*.  for example, `A function to load tile images.` is not a complete sentence and shouldn't be terminated with any punctuation.  i don't care enough to make you revert it, but i probably will at some later point.

Reply:
I broke them out for only one reason.  They had previously been omitted from the comment docs and I assume there may have been a thought process behind that and I wanted to respect it.  

Regarding this and the periods and sentences, etc., I struggled with all the changes to the comment docs.  I wanted to update the comment docs to match my changes, but I didn't want to change any more than necessary.  I tried to update only the parts that I had touched.  In doing so, most of my intent was to do so with as much consistency as possible.  In one or two cases, I may have thought I was making a correction as well.  I am truly not offended if you want to dictate any kind of changes to the comment docs for this or any other reason.  

I didn't see another place where the optional parameters were not broken out to a separate section.  I only saw a place where they were positional / non-optional parameters, which would be consistent with what I did here (I thought :) )

Reply:
there is no reason to change existing comments unless the code under them changes, context changes, etc.  any random aesthetic changes just creates churn and needless discussions like this.  please remove the `Optional named parameters:` heading as its not needed and confusing.  if you "struggled" with adding comments, perhaps the best solution would have been to leave the existing ones alone, and use the existing style for your own.  or not.  but not adding unneeded changes would be great.

Reply:
I've removed the literary corrections and reverted to the previous style.  The new comment docs are now consistent with that.